tasks/install_acmesh.yml

```diff
@@ -35,14 +35,31 @@
   changed_when: false
   when: acme_install_path.stat.exists
 
-- name: Create cron job
+- name: Create GCloud cron job
+  cron:
+    name: Acme.sh GCloud certificate renew
+    minute: '0'
+    hour: '2'
+    user: root
+    job: "GCLOUDSDK_CORE_PROJECT={{ item.project }} {{ acme_paths['install_path'] }}/acme.sh --renew --domain {{ item.domain }} --home {{ acme_paths['install_path'] }} &>> {{ acme_paths['log_file'] }}"
```

Comment:
Are we sure about this env variable? Shouldn't we use the same `CLOUDSDK_ACTIVE_CONFIG_NAME` variable instead?

Also yeah, it's probably not `GCLOUDSDK` either way

Reply:
We are, because `CLOUDSDK_ACTIVE_CONFIG_NAME` just says what config file should be used, but the projects are in the same file.

Yeah, just a typo or brain fart it should be `CLOUDSDK...` instead of `GCLOUDSDK`

Reply:
Let's try it with `CLOUDSDK_CORE_PROJECT` then